platforms/pf-robot-motion-control/motion_control.cpp

```diff
@@ -623,6 +629,24 @@ void pf_send_pb_state(void)
     pf_get_canpb().send_message(state_uuid, &pb_state);
 }
 
+void pf_send_pb_telemetry(void)
```

Comment:
Why not passing telemetry type as argument ? 🤔
Because it seems quite complex to first set telemetry type and then send it in an other function. Race condition could occur in the future with other type of telemetry. 
I think protecting the function with a mutex could also be useful. What do you think ?

Reply:
Refactored, we are good here.